slack_today_i_did/parser.py

```diff
@@ -143,11 +221,14 @@ def mismatching_args_messages(action, annotations, args) -> str:
 def mismatching_types_messages(action, annotations, args) -> str:
     messages = []
 
-    for ((arg, arg_type), (arg_name, annotation)) in zip(args, annotations.items()):  # noqa: E501
-        if arg_type != annotation:
+    for (arg, (arg_name, annotation)) in zip(args, annotations.items()):
+        if annotation != Any and not isinstance(annotation, TypeVar):
```

Comment:
When the annotation is something in particular, but isn't one of the type variables, then we want to skip it.

In what cases does this apply?